sites/partners/src/components/listings/PaperListingForm/sections/RankingsAndResults.tsx

```diff
@@ -119,6 +119,32 @@ const RankingsAndResults = ({ listing }: RankingsAndResultsProps) => {
         )}
         {reviewOrder === "reviewOrderLottery" && (
           <>
+            {process.env.showLottery && (
+              <Grid.Row columns={2} className={"flex items-center"}>
+                <Grid.Cell>
+                  <p className={`field-label m-4 ml-0`}>{t("listings.lotteryOptInQuestion")}</p>
```

Comment:
This Tailwind piece is copy pasta and overhauling this file would be too significant for this scope